src/ontology/mondo-edit.obo

```diff
@@ -61627,6 +61656,7 @@ xref: DOID:5683 {source="MONDO:equivalentTo"}
 xref: GARD:15010 {source="Orphanet:145"}
 xref: ICD10CM:C50 {source="MONDO:relatedTo", source="Orphanet:145/nd", source="Orphanet:145/attributed", source="Orphanet:145"}
 xref: ICD10CM:C56 {source="Orphanet:145/nd", source="Orphanet:145/attributed", source="Orphanet:145"}
+xref: icd11.foundation:1258896144 {source="MONDO:equivalentTo", source="Orphanet:145", source="https://orcid.org/0000-0002-4142-7153"}
```

Comment:
Not an expert on this part of things, but these kinda refs look like what I expected to see!